src/components/CheckableButton/CheckableButton.scss

```diff
@@ -43,10 +43,6 @@ $chekbox-label-margin: rem(20px);
     background: var(--p-action-secondary-pressed);
   }
 
-  @media (-ms-high-contrast: active) {
-    border: border-width(base) solid;
-  }
```

Comment:
Do we need the high contrast outline here?

Reply:
I removed it as it didn't add any visual changes 😄